src/main/java/com/cloudbees/jenkins/plugins/amazonecs/ECSTaskTemplate.java

```diff
@@ -70,6 +70,11 @@
     @CheckForNull
     private final String label;
 
+    /**
+     * Number of nodes to retain on standby
+     */
+    private final int minRetainedNodes;
```

Comment:
this setting gets persisted per-agent, so it inherits the number at the time the agent was launched. this causes different agents to honour/not honour the actual threshold. must look for a better way to configure this, maybe a https://javadoc.jenkins.io/hudson/model/labels/LabelAtomProperty.html?